packages/website/pages/docs/reference/prover-market-page.mdx

```diff
@@ -22,5 +22,6 @@ You can edit the page by clicking [here](https://github.com/taikoxyz/taiko-mono/
 | ZKPool (HTTPS) | https://taiko-a5-prover.zkpool.io       | 10 wei      |
 | Davaymne       | http://pool-1.taikopool.xyz             | 10 wei      |
 | Web3Cript      | http://taiko.web3cript.xyz:9876         | 10 wei      |
+| Web3Cript      | http://ttko.web3cript.xyz:9876          | 10 wei      |
```

Comment:
what is the difference between these two? is it means to be an updated url name? should we just have one?

Reply:
I make available another prover that I have, so I can test the CPUs of both machines well, they are two different provers, thx!

Reply:
can we please rename it then? perhaps "Web3Cript 1" and "Web3Cript 2"? because i believe the names should all be unique :)

otherwise, you can just expose one endpoint, and do a round robin / load balance to test both of them equally. in either case, we should keep the names unique.

Reply:
Of course, rename the names without problem and excuse my lack of explicitness.
Yeah, I have been doing that and since I still don't see clearly which one works better, I want to have both so I can check in real time how both are progressing.

Reply:
ok i will rename them as such, thanks.